controllers/storagecluster/external_resources.go

```diff
@@ -326,6 +344,82 @@ func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createStorageClassClaim(instance *ocsv1.StorageCluster) error {
+
+	for _, desiredStorageClassClaim := range getDefaultStorageClassClaims(instance) {
+
+		err := controllerutil.SetOwnerReference(instance, desiredStorageClassClaim, r.Client.Scheme())
+		if err != nil {
+			return err
+		}
+
+		err = r.Client.Create(context.TODO(), desiredStorageClassClaim)
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return err
+		}
+
+	}
+
+	return nil
+}
+
+func (r *StorageClusterReconciler) deleteStorageClassClaim(instance *ocsv1.StorageCluster) error {
```

Comment:
rename function to `deleteDefaultStorageClassClaims`

Reply:
done